it/uk/gov/hmrc/minorentityidentificationfrontend/assets/TestConstants.scala

```diff
@@ -302,15 +316,38 @@ object TestConstants {
 
   def testBVCtUtrJson(utr: String): JsObject = Json.obj("ctUtr" -> utr)
 
-  def testCreateBusinessVerificationJourneyJson(utrJson: JsObject,
-                                                continueUrlForBVCall: Call,
-                                                journeyConfig: JourneyConfig): JsObject =
+  def testCreateBusinessVerificationTrustJourneyJson(utrJson: JsObject,
```

Comment:
We could write like below ... a part from avoid code duplication in one spot we can see that trust is different from UA only for the presence of entity type and I dont think we loose readability ... I think we gain it a little bit more.
Again ... you can leave as it is now. Up to you
<img width="1330" alt="Screenshot 2022-03-21 at 12 24 08" src="https://user-images.githubusercontent.com/4958044/159261489-0a0a1a1a-6c6f-4fc8-aeb7-a14f7ce22b97.png">

Reply:
Fixed thank you.